src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IInterpolatedStringExpression.cs

```diff
@@ -25,8 +25,11 @@ public void M()
 }
 ";
             string expectedOperationTree = @"
-IInterpolatedStringExpression (OperationKind.InterpolatedStringExpression, Type: System.String) (Syntax: '$""""')
-  Parts(0)
+IArgument (ArgumentKind.Explicit, Matching Parameter: value) (OperationKind.Argument) (Syntax: '$""""')
```

Comment:
This feels weird. We have an ArgumentSyntaxNode that maps to an IArgument. It feels that the InterpolatedStringExpression should match with the IInterpolatedStringExpression. I think we should special case arguments to ensure IArgument is only returned for ArgumentSyntaxNode. @dotnet/analyzer-ioperation thoughts?

Reply:
I think it is a bug in factory. factory should create IArgument with ArgumentNode since it must be there.